charts/karpenter/values.yaml

```diff
@@ -18,3 +18,12 @@ webhook:
   tolerations: []
   affinity: {}
   image: "public.ecr.aws/karpenter/webhook:v0.4.2@sha256:2d004cd277209242d7e886308be419f86107811b4d0f1ed21c908b77e284202b"
+defaultProvisioner:
+  create: true
```

Comment:
Can you use the non-nil-ness of the default provisioner field to decide to create this?

Reply:
Generally, helm charts use an explicit `create` variable, so I'd prefer to keep this. It would be awkward on the CLI using `--set` with the default values.yaml if we just use the truthiness of defaultProvisioner.